src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -139,12 +139,13 @@ internal void SetBuffer(byte[] buffer, int offset, int count)
         {
             _bufferedContent = new MemoryStream(buffer, offset, count, writable: false, publiclyVisible: true);
         }
-        
+
         internal bool TryGetBuffer(out ArraySegment<byte> buffer)
         {
+            buffer = default(ArraySegment<byte>);
```

Comment:
Was getting a compiler error here when compiling locally with roslyn.  Double check that this builds in a VS2015 command-prompt.